### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by aays**
Not complete yet! Needs a bit of fleshing out still.

Addresses UofTCoders/paper-rcourse#38.

### CODE REVIEW

**Comment by lwjohnst86** [comment]
Looks good so far :)

File: README.md [hunk 1]
```diff
@@ -7,4 +7,50 @@ Course in Ecology and Evolutionary Biology
...
 
+
+## Description
+
+Quantitative Methods in R for Biology is a course aimed at undergraduates at a third year level or above. 
+The course covers statistics and data analysis for ecology and reproducible quantitative methods in R. 
+
+Statistical analysis, modelling, simulation, and data analysis are essential skills for applying 
+ecology concepts to data. This course is designed to meet a growing demand for reproducible, openly accessible, 
+analytically thorough, and well documented science. Students will learn to develop ecological population models,
```
Comment (mbonsma) [inline README.md]: s/well documented/well-documented

File: README.md [hunk 2]
```diff
@@ -7,4 +7,50 @@ Course in Ecology and Evolutionary Biology
...
+## Instructional Design
+
+The lectures in this course are designed to be presented using a participatory live-coding approach.
+This involves an instructor typing and running code in [RStudio](https://www.rstudio.com/) in front of the class, while the class follows
+along using their own computers. Challenges are interspersed in the lesson material, allowing students to 
+collaboratively work on smaller coding problems for a few minutes. All lesson materials are provided ahead
+of time on the course website for students to refer to during lectures.
+
+The bulk of the course's assessment structure involves weekly assignments. These assignments
+are primarily code-based, and are designed to also be completed in RStudio using the R Markdown format.
```
Comment (mbonsma) [inline README.md]: no comma after `code-based`

File: README.md [hunk 3]
```diff
@@ -7,4 +7,50 @@ Course in Ecology and Evolutionary Biology
...
+
+At the end of the course, students undertake a group project, wherein they attempt to address a scientific
+question by applying techniques learned over the course to open ecological data. At the end of the semester,
+groups present their work in a conference-style presentation, and submit a report in the style of a scientific paper. 
+
+## Lecture Content
+
+The course's lesson material is broadly subdivided into three main topics:
```
Comment (mbonsma) [inline README.md:41]: s/three/four 

although Sara likes 'three' as a nice round number, so maybe we could combine 2 and 3 here like you had before?

File: README.md [hunk 4]
```diff
@@ -7,4 +7,50 @@ Course in Ecology and Evolutionary Biology
...
+
+1. Introductory R (Lectures 1-5)
+    * Introduces students to the R programming language, with a focus on 
+      data wrangling and visualization.
+2. Statistical analysis (Lectures 6-10)
+    * Introduces concepts such as regression, principal component analysis, and statistical models
```
Comment (mbonsma) [inline README.md]: Period at the end

File: README.md [hunk 5]
```diff
@@ -7,4 +7,50 @@ Course in Ecology and Evolutionary Biology
...
+3. Numerical modelling (Lectures 11-12)
+    * Introduces students to numerically modelling populations in R
```
Comment (mbonsma) [inline README.md]: period

File: README.md [hunk 6]
```diff
@@ -7,4 +7,50 @@ Course in Ecology and Evolutionary Biology
...
+3. Reproducible science (Lectures 13-15)
+    * Prepares students for project work period and introduces methods for reproducible science (GitHub, R Markdown)
```
Comment (mbonsma) [inline README.md]: Period

**Review by mbonsma — APPROVED** [review]

**Comment by aays** [comment]
Should be good to go now!

File: README.md [hunk 7]
```diff
@@ -7,4 +7,59 @@ Course in Ecology and Evolutionary Biology
 [![Build Status](https://travis-ci.org/UofTCoders/rcourse.svg?branch=master)](https://travis-ci.org/UofTCoders/rcourse)
 [![DOI](https://zenodo.org/badge/97400494.svg)](https://zenodo.org/badge/latestdoi/97400494)
 
+
+## Description
+
+Quantitative Methods in R for Biology is a course aimed at undergraduates at a third year level or above. 
+The course covers statistics and data analysis for ecology and reproducible quantitative methods in R. 
+
+Statistical analysis, modelling, simulation, and data analysis are essential skills for applying
```
Comment (James-S-Santangelo) [inline README.md]: Change first sentence to: "Statistical analysis, modelling, and data simulation are essential skills for ecologists and evolutionary biologists."

File: README.md [hunk 8]
```diff
@@ -7,4 +7,59 @@ Course in Ecology and Evolutionary Biology
...
+## Description
+
+Quantitative Methods in R for Biology is a course aimed at undergraduates at a third year level or above. 
+The course covers statistics and data analysis for ecology and reproducible quantitative methods in R. 
+
+Statistical analysis, modelling, simulation, and data analysis are essential skills for applying 
+ecology concepts to data. This course is designed to meet a growing demand for reproducible, openly accessible, 
+analytically thorough, and well-documented science. Students will learn to develop ecological population models, 
+analyze data, and document their research using the R programming language. No prior programming experience is required.
+
```
Comment (James-S-Santangelo) [inline README.md:21]: Maybe add something about how students will learn to clean, manipulate, and visualize "big data", which is increasingly generated and used in a range of scientific disciplines.

**Review by James-S-Santangelo — CHANGES_REQUESTED** [review]

**Review by James-S-Santangelo — APPROVED** [review]

**Pull request merged**